packages/core/src/utils/htmlValidationUtil.ts

```diff
@@ -0,0 +1,18 @@
+import cheerio from 'cheerio';
+import * as logger from './logger';
+
+function validateAndLogTableHasTbody(rootNode: cheerio.Root, path: string) {
+  const tables = rootNode('table');
+  for (let i = 0; i < tables.length; i += 1) {
+    const table = rootNode(tables[i]);
+    if (table.find('tbody').length === 0) {
+      // eslint-disable-next-line max-len
```

Comment:
Don't disable this. Just split the line up

Reply:
Modified as suggested